fuel-tests/tests/helpers.rs

```diff
@@ -0,0 +1,153 @@
+use fuel_core::chain_config::{ChainConfig, CoinConfig, ContractConfig, StateConfig};
+use fuel_core::service::{Config, FuelService};
+use fuel_gql_client::client::FuelClient;
+use fuel_tx::Transaction;
+use fuel_vm::prelude::*;
+use itertools::Itertools;
+use rand::{rngs::StdRng, Rng, SeedableRng};
+use std::collections::HashMap;
+
+/// Helper for configuring the genesis block in tests
+pub struct TestSetupBuilder {
+    pub rng: StdRng,
+    pub contracts: HashMap<ContractId, ContractConfig>,
+    pub initial_coins: Vec<CoinConfig>,
+    pub min_gas_price: u64,
+    pub min_byte_price: u64,
+}
+
+pub struct TestContext {
+    pub rng: StdRng,
+    pub client: FuelClient,
+}
+
+impl TestSetupBuilder {
+    pub fn new(seed: u64) -> TestSetupBuilder {
+        Self {
+            rng: StdRng::seed_from_u64(seed),
+            ..Default::default()
+        }
+    }
+
+    /// setup a contract and add to genesis configuration
+    pub fn setup_contract(&mut self, code: Vec<u8>) -> (Salt, ContractId) {
```

Comment:
```suggestion
    pub fn setup_contract(&mut self, code: Vec<u8>, balances: Option<HashMap<AssetId, u64>>) -> (Salt, ContractId) {
```

Reply:
Changed to now accept an Option of a Vec of Tuples